cypress/e2e/data-import/settings/duplicate-an-existing-match-profile.cy.js

```diff
@@ -0,0 +1,66 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import getRandomStringCode from '../../../support/utils/genereteTextCode';
+import { EXISTING_RECORDS_NAMES } from '../../../support/constants';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import Users from '../../../support/fragments/users/users';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import MatchProfileView from '../../../support/fragments/data_import/match_profiles/matchProfileView';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const matchProfile = {
+      profileName: `C2340 autotest match profile ${getRandomStringCode(8)}`,
+      incomingRecordFields: {
+        field: '001',
+      },
+      existingRecordFields: {
+        field: '001',
+      },
+      matchCriterion: 'Exactly matches',
+      existingRecordType: EXISTING_RECORDS_NAMES.MARC_BIBLIOGRAPHIC,
+    };
+
+    const duplicatedMatchProfile = {
+      profileName: `C2340 autotest duplicate match profile ${getRandomStringCode(8)}`,
+      matchCriterion: 'Existing value contains incoming value',
+    };
+
+    const calloutMessage = `The match profile "${duplicatedMatchProfile.profileName}" was successfully created`;
+    const calloutErrorMessage = `Match profile '${matchProfile.profileName}' already exists`;
```

Comment:
The error message here is different from the one which is in the test case, please clarify this with the functional QA engineer.

Reply:
fixed